modules/network-observability-ebpf-rule-flow-filter.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ebpf-flow-rule-filter_{context}"]
+= eBPF flow rule filter
+You can use rule-based filtering to control the volume of packets cached in the eBPF flow table. For example, a filter can specify that only packets coming from port 100 should be recorded. Then all packets which do not match the filter are not cached and only the ones that match are cached.
+
+== How ingress and egress traffic filtering works
+CIDR notation allows for the efficient representation of IP address ranges by combining the base IP address with a prefix length. For both ingress and egress traffic, the source IP address is first used to match filter rules configured with CIDR notation. If the source IP address isn't configured, then the destination IP is used to match filter rules configured with CIDR notation. 
```

Comment:
what does it mean by `the source IP address isn't configure` this is the pkt srcIP no configuration here the only config here is the filter rule CIDR 
we 1st match packet srcIP against the CIDR if match we proceed if no match we check against pkt dstIP

Reply:
Yeah you're right here. I honestly don't know why I wrote `configure` here, I can see how it needs to be changed to `match`